src/lib/core/ripple/ripple.ts

```diff
@@ -87,8 +84,18 @@ export class MdRipple implements OnChanges, OnDestroy {
   }
 
   /** Launches a manual ripple at the specified position. */
-  launch(pageX: number, pageY: number, config = this.rippleConfig) {
-    this._rippleRenderer.fadeInRipple(pageX, pageY, config);
+  launch(pageX: number, pageY: number, config = this.rippleConfig): RippleRef {
+    return this._rippleRenderer.fadeInRipple(pageX, pageY, config);
+  }
+
+  /** Fades out all currently showing ripple elements. */
+  fadeOutAll() {
+    // Iterate in reverse, to avoid issues with the `fadeOut` method that will immediately remove
+    // items from the array.
+    let i = this._rippleRenderer.activeRipples.length;
+    while (i--) {
```

Comment:
Hmm not sure. I didn't want to put all of the logic to the renderer. The render is actually just responsible for the DOM manipulation. Any other arguments why we should move it to the renderer as well?